rotkehlchen/tests/unit/test_eth2.py

```diff
@@ -195,6 +199,41 @@
 ]
 
 
+def test_eth2_equivalent_eth_balances(database: DBHandler):
+    """Test that the balances of ETH and ETH2 are same."""
+    balances = [
+        DBAssetBalance(
+            category=BalanceType.ASSET,
+            time=Timestamp(1488326400),
+            asset=A_ETH,
+            amount='10',
+            usd_value='4517.4',
+        ),
+        DBAssetBalance(
+            category=BalanceType.ASSET,
+            time=Timestamp(1488326400),
+            asset=A_ETH2,
+            amount='10',
+            usd_value='4517.4',
+        ),
+    ]
+    database.add_multiple_balances(balances)
+
+    # Test query_timed_balances for ETh
+    with patch(
+            'rotkehlchen.db.dbhandler.DBHandler.get_settings',
+            return_value=DBSettings(eth_equivalent_eth2=True),
+    ):
+        eth_balances = database.query_timed_balances(A_ETH)
+        assert len(eth_balances) == 2
+
+    eth2_balances = database.query_timed_balances(A_ETH2)
+    assert len(eth2_balances) == 1
```

Comment:
ETH2 should dissapear as an asset when this setting is on.
```suggestion
    eth2_balances = database.query_timed_balances(A_ETH2)
    assert len(eth2_balances) == 0
```